config/application.rb

```diff
@@ -28,5 +28,14 @@ class Application < Rails::Application
 
     # Autoloadable lib directories with classes and modules.
     config.autoload_paths += %W(#{config.root}/lib)
+
+    # Spristupnenie konstant
+    config.before_configuration do
+      env_file = File.join(Rails.root, 'config', 'local_env.yml')
+      YAML.load(File.open(env_file)).each do |key, value|
```

Comment:
Hmmm, ale toto naozaj nie su hesla, secret kluce do APIs a pod., ktore by sme (teraz|raz) chceli znepristupnit pre repozitar... (Aj ked vahy do skore by sme teoreticky mohli chciet skryvat pred studentmi, ktori to skore ziskavaju, pozdravujem ak niekto cita :) )

Skor by sme to chceli uplne naopak, ked si niekto vyklonuje repozitar, aby nemusel zistovat, ako to kde kedy bolo na ake hodnoty nastavene, ale aby mal rovnake hodnoty ako vsetci ostatni. Aj ked na druhu stranu, mozeme chciet experimentovat s roznymi vahami na stagingu/sandboxe/production a potom ma zmysel nemat to v kode, ale v YAML, ktory moze byt v nasadeni nahradzovany symlinkou (ha, alebo v DB).

Podla mna by najviac sedeli dva pristupy:

1. Povedat, ze to je nastavenie kodu ako takeho => potom by mi to sedelo mat to niekde v `lib`, kde sa to zaroven bude aj pocitat (urcite by to nemal ratat Controller, Model mozno).

2. Povedat, ze to je nastavenie konkretneho nasadenia => potom by mi to sedelo mat to v DB ako nastavenie setupu (vid nastavenie odporucaca v starom ALEFe) a ked niekto prevezme dump zo stagingu/sandboxu/production, tak to ma nastavene na presne rovnake hodnoty, ako to bolo tam a moze na tom robit datalytics kolko len chce.